dags/trigger_k8s_cronjob.py

```diff
@@ -22,9 +20,24 @@ def get_cronjob(cronjob_name, namespace):
             "Exception when calling BatchV1Api->list_namespaced_cron_job: %s\n" % e)
     return False
 
-# Creates a job from a cronjob job_template
-
+# Deletes all non-active jobs
+def remove_finished_jobs(namespace):
+    configuration = client.Configuration()
+    batch = client.BatchV1Api(client.ApiClient(configuration))
+    try:
+        jobs = batch.list_namespaced_job(namespace).items
+        for job in jobs:
+            if job.status.active == 0:
+                try:
+                    batch.delete_namespaced_job(job.metadata.name, namespace)
```

Comment:
this might silently delete jobs that failed, but we would have kept these in airflow notifications and logs?

Reply:
yes this will delete failed jobs. Do we want to only delete completed jobs? And manually delete failed jobs?

Reply:
...maybe? @matthieu-foucault any opinion on that?
The other thing is that if we have a DAG that triggers multiple cronjobs we'll be deleting previous pods as each step of the DAG completes, so we wouldn't even keep the job history of a full DAG call